CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Before we Start
+
+We want to thank anyone who wishes to take time and to support this project.
+
+This project was made with the benefit of the Magshimim Next Community in mind. \
+Wether it's by giving them a platform to upload and improve their CVs and wether it's to try and play with some web development and penetration testing.
+
+And with that in mind:
+
+> [!IMPORTANT]
+> We have decided to only allow members of the community to contribute to the project.
+
+## For non-members
+
+Again, we really appreciate your time and dedication to the open source community and we are sure your skills can be much more valuable in a globaly available open source project! Keep up the motivation!
+
+Also note that the repo itself will still remain public to allow other people to use it as a template or a base for their own forks to be used as anything they might want!
+
+## For members
+
+Thank you for wanting to contribute to the project and the community!
+
+First, if you have any question, contact us from the community group and we'll gladly help!
+
+### Feedback and Suggestions
+
+TODO: decide how we want to accept feedback and more suggestions
+
+### How to start
+
+Grab an issue you think you can manage, assign it to yourself, and create a fork for it!
+
+### Conventions
+
+TODO: enter conventions
```

Comment:
@YechiamW3 should we add something here or just delete it?

Reply:
If we want to write conventions we'd probably want a separate file; for now I think we should keep it empty and add later if we see necessary because keeping the conventions in check via the CRs is good enough for now, in my opinion.
Anyway, you can assign to me an issue to write such a file.

Reply:
Added #177

Reply:
Resolving this but updated the todo